README.md

```diff
@@ -0,0 +1,94 @@
+# gr-stellarstation: GNURadio blocks for interacting with StellarStation
+
+gr-stellarstation is an out-of-tree GNURadio module that provides blocks for easily integrating with the [StellarStation](https://www.stellarstation.com/) API in GNURadio flowgraphs.
+
+## Installation
+
+### PyBOMBS
+
+PyBOMBs support is currently under construction.
+
+### Source Installation
+
+gr-stellarstation, like most OOT modules, can be built from source using CMake.
+
+First, we need to install gr-stellarstation's sole dependency, [Conan](https://conan.io/). The simplest way to do this is through `pip`:
+
+```bash
+$ pip install conan
+```
+
+For alternative methods to install Conan, see the [Conan documentation](https://docs.conan.io/en/latest/installation.html).
+
+After installing Conan, run the following commands to add the relevant Conan recipes:
+
+```bash
+$ conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
+$ conan remote add inexorgame https://api.bintray.com/conan/inexorgame/inexor-conan
+$ conan remote add stellarstation https://api.bintray.com/conan/infostellarinc/stellarstation-conan
+```
+
+Proceed to install gr-stellarstation normally with CMake:
+
+```bash
+$ git clone https://github.com/infostellarinc/gr-stellarstation.git
+$ cd gr-stellarstation
+$ mkdir build
+$ cmake ..
```

Comment:
After installing cppunit and Doxygen, I still have errors:

```
CMake Error at CMakeLists.txt:149 (find_package):
  By not providing "FindGnuradio.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Gnuradio",
  but CMake did not find one.

  Could not find a package configuration file provided by "Gnuradio"
  (requested version 3.7.2) with any of the following names:

    GnuradioConfig.cmake
    gnuradio-config.cmake

  Add the installation prefix of "Gnuradio" to CMAKE_PREFIX_PATH or set
  "Gnuradio_DIR" to a directory containing one of the above files.  If
  "Gnuradio" provides a separate development package or SDK, be sure it has
  been installed.

```

It's OK if users shares knowledge around it.

Reply:
Did you install GNURadio properly before this? I think your errors

Reply:
Yeah, but maybe only binaries are installed by apt-get . Never mind this comment since it looks my environmental issue.